server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -33,6 +33,18 @@ import {
 } from "@fluidframework/server-services-telemetry";
 import { getBooleanFromConfig, getNumberFromConfig } from "./configUtils";
 
+interface IKeylessTokenClaims extends ITokenClaims {
+	/**
+	 * Identifies if the token is for Keyless Access or not.
+	 */
+	fluidRelayKeylessAccess?: boolean;
```

Comment:
Since this is in R11s, which is not necessarily Fluid Relay, could we change this to something more generic like `useKeylessAccess` or `enableKeylessAccess` or `isKeylessToken`?